base_wamas_ubl/lib/wamas/test_utils.py

```diff
@@ -29,6 +41,56 @@ def testDict2wamas(self):
             output = dict2wamas(ast.literal_eval(str_dict_input), telegram)
             self.assertEqual(output, expected_output)
 
+    def testGetAddressNamePicking(self):
+        infile = file_open(
+            file_path("tests/samples/UBL2WAMAS-SAMPLE_AUSK_AUSP-DESPATCH_ADVICE.xml")
+        ).read()
+        dict_item = Dotty(xmltodict.parse(infile))
+        address_elements = get_address_elements(dict_item)
+        self.assertEqual(
+            address_elements,
+            {
+                "ContactName": "Chester Reed",
+                "PartyName": "YourCompany",
+                "Department": "Department_1",
+                "StreetName": "250 Executive Park Blvd, Suite 3400",
+                "AdditionalStreetName": "AdditionalStreetName_1",
+            },
+        )
+        self.assertEqual(get_Adrs_Name(address_elements), "Chester Reed")
+        self.assertEqual(get_Adrs_Name2(address_elements), "YourCompany")
+        self.assertEqual(get_Adrs_Name3(address_elements), "Department_1")
+        self.assertEqual(
+            get_Adrs_Name4(address_elements), "250 Executive Park Blvd, Suite 3400"
+        )
+        self.assertEqual(get_Adrs_Adr(address_elements), "AdditionalStreetName_1")
+
+    def testGetAddressNameFromDict(self):
+        dict_data = {
```

Comment:
we don't need this, just feed `address_elements` directly (we don't need to test `get_address_elements` twice)